images/csp_violation_report_service/app/databases/seeds/database_seeder.py

```diff
@@ -0,0 +1,10 @@
+"""Base Database Seeder Module."""
+from masoniteorm.seeds import Seeder
+
+from .user_table_seeder import UserTableSeeder
+
+
+class DatabaseSeeder(Seeder):
+    def run(self):
+        """Run the database seeds."""
+        # self.call(UserTableSeeder)
```

Comment:
I'm guessing this is only something we'd need for local dev?

Reply:
preview app as well but i don't have plans for that at the moment even tho masonite makes it super simple by supporting sqlite databases.